src/404.md

```diff
@@ -0,0 +1,13 @@
+---
+permalink: 404.html
+layout: layouts/page.html
+title: 404 Not Found
+---
+
+## 404 No Bytes Found
+
+<code class="tty">
+    <pre><span class="tty-username">admin</span>&commat;<span class="tty-hostname">bytezone.dev</span> &dollar; ls -l -h /var/www/static</pre>
+    <pre>lrwxrwx--- 1 root www    9 Oct  4 09:58 <span class="tty-broken-link">current.html</span> &longrightarrow; <span class="tty-broken-link">page.html</span></pre>
+    <pre>-rw-r----- 1 root www 2.4K Oct  4 09:57 <a href="/">index.html</a></pre>
```

Comment:
I had no idea this was a link at all until I looked here.  We need to make this more clear somehow.

Reply:
The normal page navigation still exists. I included the href as an easter egg.

Reply:
Maybe just an underline?

Reply:
Added underline & silver text color. Which is a very slight color change.